docs/DeveloperGuide.md

```diff
@@ -171,6 +171,46 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Tag Management
+
+This section will describe tag management in the address book as well as the features implemented.
+
+### Centralising Tags in the Address Book
+
+In the previous implementation, all `Tag`s are independent of one another despite having the same tag names. Hence, centralising the tags enable the user to easily manipulate them as well as easily searching for `Person`s that contain the tag.
+This is done by creating a `UniqueTagList` within `AddressBook` which will store all tags that were created by the user. Whenever a command relating to `Tag` is executed, it will not only apply the changes to the `Tag` in the `UniqueTagList` but will propagate these changes to the relevant `Person`s who contain the specified tag.
+Hence, all operations relating to the `Tag`s are done at the `AddressBook` level to ensure that the `Tag`s and `Person`s are properly synchronised.
```

Comment:
Once again, `hence` does not really make sense in this context, you should drop it.
```suggestion
All operations relating to the `Tag`s are done at the `AddressBook` level to ensure that the `Tag`s and `Person`s are properly synchronised.
```

Reply:
Fixed in 61207721